bundle-workflow/python/test.py

```diff
@@ -8,13 +8,18 @@
 from git.git_repository import GitRepository
 from test_workflow.test_cluster import LocalTestCluster
 from test_workflow.integ_test_suite import IntegTestSuite
+from paths.script_finder import ScriptFinder
 
 parser = argparse.ArgumentParser(description = "Test an OpenSearch Bundle")
 parser.add_argument('manifest', type = argparse.FileType('r'), help="Manifest file.")
 parser.add_argument('--workdir', type = pathlib.Path, help="Specify a working directory. If this is not specified then a temporary directory will be used. This option is mostly useful for debugging - it will leave the contents of the directory after the tool finishes, whereas a temporary directory will be automatically deleted.")
 args = parser.parse_args()
 
 manifest = BundleManifest.from_file(args.manifest)
+component_scripts_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../scripts/bundle-build/components')
+default_scripts_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../scripts/bundle-build/standard-test')
```

Comment:
This doesn't seem to be a path in the repo. The `standard-test` is in /tools/standard-test/

Reply:
You are not missing something. I forgot to add that directory.

Fixed now.

Reply:
Good 👀